src/Kusto/test/Add-AzKustoDatabasePrincipal.Tests.ps1

```diff
@@ -0,0 +1,22 @@
+$loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+if (-Not (Test-Path -Path $loadEnvPath)) {
+    $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+}
+. ($loadEnvPath)
+$TestRecordingFile = Join-Path $PSScriptRoot 'Add-AzKustoDatabasePrincipal.Recording.json'
+$currentPath = $PSScriptRoot
+while(-not $mockingPath) {
+    $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+    $currentPath = Split-Path -Path $currentPath -Parent
+}
+. ($mockingPath | Select-Object -First 1).FullName
+
+Describe 'Add-AzKustoDatabasePrincipal' {
+    It 'AddExpanded' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
```

Comment:
Why are test cases all skipped?

Reply:
we are testing the scenario in the New-AzKustoDatabasePrincipalAssignment file.

Reply:
Can you just delete this file?

Reply:
done